From 51319a6989c7b0cf8e1909b256d85accce981df4 Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Sun, 14 Aug 2016 11:30:15 +0300
Subject: [PATCH 0403/1345] fix: pp3: fix message unsupported autoneg-restart

commit  2259e5cc1724988003a7facb83c45e34828280bf from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

JIRA SYSTEMSW-2663
On "ethtool -r nicN" command fix incorrect message
  mv_pp3_gop_autoneg_restart: on supported for port mode (0)#
with correct one:
  gop port N: autoneg-restart not supported for mode=0 (XAUI/RXAUI)
  #

Change-Id: I78dcbe966b560855ad09b0fb95ce3e691109968a
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31911
Reviewed-by: Dmitri Epshtein <dima@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Yelena Krivosheev <yelena@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/pp3/gop/mv_gop_if.c |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/marvell/pp3/gop/mv_gop_if.c b/drivers/net/ethernet/marvell/pp3/gop/mv_gop_if.c
index a8b2925..d29fc97 100644
--- a/drivers/net/ethernet/marvell/pp3/gop/mv_gop_if.c
+++ b/drivers/net/ethernet/marvell/pp3/gop/mv_gop_if.c
@@ -662,10 +662,12 @@ int mv_pp3_gop_autoneg_restart(int port_num)
 	break;
 	case MV_PORT_XAUI:
 	case MV_PORT_RXAUI:
-		pr_err("%s: on supported for port mode (%d)", __func__, gop_ports[port_num].port_mode);
+		pr_err("gop port %d: autoneg-restart not supported for mode=%d (XAUI/RXAUI)\n",
+		       port_num, gop_ports[port_num].port_mode);
 		return -1;
 	default:
-		pr_err("%s: Wrong port mode (%d)", __func__, gop_ports[port_num].port_mode);
+		pr_err("gop port %d: autoneg-restart wrong mode=%d\n",
+		       port_num, gop_ports[port_num].port_mode);
 		return -1;
 	}
 	return 0;
-- 
1.7.9.5

